cve_bin_tool/checkers/libevent.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for bash
+https://www.cvedetails.com/product/32303/Libevent-Project-Libevent.html?vendor_id=15590
+"""
+from . import Checker
+
+
+class LibeventChecker(Checker):
+    CONTAINS_PATTERNS = [r"Abstract asynchronous event notification library"]
+    FILENAME_PATTERNS = [r"libevent-2.1.so."]
+    VERSION_PATTERNS = [
+        r"([0-9]+\.[0-9]+\.[0-9]+)-stable",

```

Comment:
This isn't specific enough to libevent -- lots of other packages have `X.X.X` patterns and I would not be shocked if some of them matched with a `-stable` in there too.  Is there anything more specific you can add, or should we add libevent to the list of "signature not easy to find" checkers?

Reply:
Give me 1 - 2 days I will find a more specific pattern for it.